libs/wingc/src/jsify.rs

```diff
@@ -1819,7 +1752,8 @@ impl Visit<'_> for FreeVariableScanner {
 				// If this reference is a capturable, non-reassignable, preflight
 				// variable then it's a capture
 				if let SymbolKind::Variable(v) = lookup_result {
-					if v.type_.is_capturable() && !v.reassignable && v.phase == Phase::Preflight {
+					// TODO: if we see a reassignable or non-capturable variable, should we emit an error here or inside analyze_expr?
+					if v.phase == Phase::Preflight {
```

Comment:
@yoav-steinberg note this change - if I didn't do this, I saw some diagnostics started to disappear (about disallowing capturing reassignable values)... not sure if this is correct

Reply:
Ok, I guess `analyze_expr` created those diags. Since this is in jsification phase, it's a bit of an issue either way. We need these diags to be available in the lsp too..
@staycoolcall911 Lets have a discussion about this and see if this is P1, if we need an issue for this or what...

Reply:
https://github.com/winglang/wing/issues/2548